src/events/cd-event-list-item.vue

```diff
@@ -36,6 +32,14 @@
         {{ isFull ? $t('Full') : $t('See Details and Book') }}
       </router-link>
     </div>
+    <div v-if="isRecurring" class="cd-event-list-item__recurring-info">
+      <span class="fa fa-info-circle cd-event-list-item__recurring-info-icon"></span>
+      <span class="cd-event-list-item__recurring-info-header">{{ $t('This is a recurring event') }}</span>
+      <div class="cd-event-list-item__recurring-info-text">
+        {{ `${recurringFrequencyInfo} ${$t('at')}` }} {{ event.dates[0].startTime | cdTimeFormatter }} - {{ event.dates[0].endTime | cdTimeFormatter }},
+        {{ $t('from') }} {{ event.dates[0].startTime | cdDateFormatter }} {{ $t('to') }} {{ event.dates[event.dates.length - 1].startTime | cdDateFormatter }}
```

Comment:
I only did it this way because filters cannot be used in String interpolation and are used to format the date and time